common/contract.ts

```diff
@@ -31,9 +31,10 @@ export type Contract<T extends AnyContractType = AnyContractType> = {
   closeTime?: number // When no more trading is allowed
 
   isResolved: boolean
-  resolutionTime?: number // When the contract creator resolved the market
+  resolutionTime?: number // When the market is resolved
   resolution?: string
-
+  autoResolution: resolution
+  autoResolutionTime: number // When the market will be resolved automatically
```

Comment:
This seems like a lie because there will be many contracts for which these fields are not set.